<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformDialogBase</class>
 <widget class="QDialog" name="TransformDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>283</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QListWidget" name="transformSelector"/>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="buttonAdd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>21</width>
             <height>26</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QStackedWidget" name="transformStack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>16</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="blankPage"/>
         <widget class="QWidget" name="pageScale">
          <layout class="QGridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="ScalingStackLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Scaling</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="horizontalScaleLabel">
             <property name="text">
              <string>Horizontal:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ScrSpinBox" name="horizontalScale">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="decimals" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2" rowspan="2">
            <widget class="LinkButton" name="scaleLink">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="verticalScaleLabel">
             <property name="text">
              <string>Vertical:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ScrSpinBox" name="verticalScale">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="decimals" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageTranslate">
          <layout class="QGridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="TranslationStackLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Translation</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="translateHorizontalLabel">
             <property name="text">
              <string>Horizontal:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ScrSpinBox" name="translateHorizontal">
             <property name="minimum">
              <number>-30000</number>
             </property>
             <property name="maximum">
              <number>30000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="translateVerticalLabel">
             <property name="text">
              <string>Vertical:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ScrSpinBox" name="translateVertical">
             <property name="minimum">
              <number>-30000</number>
             </property>
             <property name="maximum">
              <number>30000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageRotate">
          <layout class="QGridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="RotateStackLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Rotate</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="rotationValueLabel">
             <property name="text">
              <string>Angle:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ScrSpinBox" name="rotationValue">
             <property name="minimum">
              <number>-360</number>
             </property>
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="decimals" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Seite">
          <layout class="QGridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="SkewStackLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Skew</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="horizontalSkewLabel">
             <property name="text">
              <string>Horizontal:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ScrSpinBox" name="horizontalSkew">
             <property name="minimum">
              <number>-89</number>
             </property>
             <property name="maximum">
              <number>89</number>
             </property>
             <property name="decimals" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2" rowspan="2">
            <widget class="LinkButton" name="linkSkew">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="verticalSkewLabel">
             <property name="text">
              <string>Vertical:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ScrSpinBox" name="verticalSkew">
             <property name="minimum">
              <number>-89</number>
             </property>
             <property name="maximum">
              <number>89</number>
             </property>
             <property name="decimals" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QGroupBox" name="RotationGroup">
           <property name="title">
            <string>Basepoint</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="BasePointWidget" name="basePoint" native="true">
              <property name="minimumSize">
               <size>
                <width>56</width>
                <height>56</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Copies</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="numberOfCopies">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="10"/>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LinkButton</class>
   <extends>QToolButton</extends>
   <header>ui/linkbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BasePointWidget</class>
   <extends>QWidget</extends>
   <header>ui/basepointwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TransformDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TransformDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
